ouroboros-consensus-cardano/README.md

```diff
@@ -7,9 +7,273 @@ particular for the Byron, Shelley and Cardano instantiations.
 
 There are also test-suites for each of the block definitions.
 
+# Consensus DB tools
+
 This package also contains two executables:
 
 * `app/db-analyser.hs`: performs different analyses on a ChainDB, for
   performance measurements or ensuring validity.
-  
+
 * `app/db-synthesizer`: builds a chain, to be used with benchmarking purposes.
+
+NB: our top level `cabal.project` enables assertions in both our local packages
+and the ones we depend on. If you build these tools from this repository, it is
+recommended that you override this locally by means of a `cabal.project.local`.
+
+## db-analyser
+
+### About
+This tool was initially developed to help Consensus debugging some issues, while the team was still working on Shelley. Later it was recognized that db-analyser might be potentially used by other teams when benchmarking / profiling some part of the code base.
+
+### Running the tool
+
+When you run db-analyser without any arguments, it will print out a nice helper message:
+
+```
+cabal build ouroboros-consensus-cardano-tools:db-analyser
+
+Missing: --db PATH COMMAND
+
+Usage: db-analyser --db PATH [--verbose]
+            [--only-immutable-db [--analyse-from SLOT_NUMBER]]
+            [--validate-all-blocks | --minimum-block-validation] COMMAND
+            [--show-slot-block-no | --count-tx-outputs |
+                --show-block-header-size | --show-block-txs-size |
+                --show-ebbs | --store-ledger SLOT_NUMBER | --count-blocks |
+                --checkThunks BLOCK_COUNT | --trace-ledger |
+                --repro-mempool-and-forge INT]
+            [--num-blocks-to-process INT]
+  Simple framework used to analyse a Chain DB
```

Comment:
This help output is outdated (e.g. `--benchmark-ledger-ops` is missing); what about removing it as it is easy to let this go stale and people can see it locally?

Reply:
Absolutely!